packages/dapp/src/App.jsx

```diff
@@ -24,55 +23,112 @@ import { ViewInstantInvoice } from './pages/instant/ViewInstantInvoice';
 import { Layout } from './shared/Layout';
 import { globalStyles, theme } from './theme';
 import { CreateInvoiceInstant } from './pages/instant/CreateInvoiceInstant';
+import '@rainbow-me/rainbowkit/dist/index.css';
+import {
+  connectorsForWallets,
+  RainbowKitProvider,
+} from '@rainbow-me/rainbowkit';
+import { configureChains, createConfig, WagmiConfig } from 'wagmi';
+import { mainnet, polygon, goerli, gnosis, polygonMumbai } from 'wagmi/chains';
+import { infuraProvider } from 'wagmi/providers/infura';
+import { publicProvider } from 'wagmi/providers/public';
+import {
+  walletConnectWallet,
+  injectedWallet,
+  rainbowWallet,
+  ledgerWallet,
+  coinbaseWallet,
+} from '@rainbow-me/rainbowkit/wallets';
+
+const APP_NAME = 'Smart Invoice';
+const PROJECT_ID = process.env.REACT_APP_WALLETCONNECT_ID;
```

Comment:
We need to communicate this change so it is added to the environments that exist.

Reply:
yes, thanks for reminder